<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptRosterMuc</class>
 <widget class="QWidget" name="OptRosterMuc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OptApplicationUI</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="ck_disableScrollbar">
     <property name="toolTip">
      <string>Hides scrollbar in the roster widget.</string>
     </property>
     <property name="text">
      <string>Disable scrollbar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_icons">
     <property name="title">
      <string>Icons</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QCheckBox" name="ck_showStatusIcons">
        <property name="toolTip">
         <string>Show icon if contact status like online, away and others.</string>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="ck_showClientIcons">
        <property name="toolTip">
         <string>Show icon of user client</string>
        </property>
        <property name="text">
         <string>Clients</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="ck_showAffiliationIcons">
        <property name="toolTip">
         <string>Show who is the boss and who is nothing</string>
        </property>
        <property name="text">
         <string>Affiliations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="ck_showAvatarIcons">
        <property name="toolTip">
         <string>Show avatars in roster</string>
        </property>
        <property name="text">
         <string>Avatars</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_avatarsSettings">
     <property name="title">
      <string>Avatar settings</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl_avatarsSize">
          <property name="text">
           <string>Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_avatarsSize">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>pt</string>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lbl_avatarsRadius">
          <property name="text">
           <string>Radius of corners</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_avatarsRadius">
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>pt</string>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="ck_avatarsAtLeft">
        <property name="toolTip">
         <string>Align avatars to the left edge of the roster. Otherwise to the right.</string>
        </property>
        <property name="text">
         <string>Left aligned</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
